src/plugins/intel_gna/src/gna_graph_compiler.cpp

```diff
@@ -2588,8 +2607,8 @@ void GNAGraphCompiler::connectOutput(InferenceEngine::CNNLayerPtr layer, void* p
 ConnectionDetails GNAGraphCompiler::connectInput(CNNLayerPtr layer,
                                                  void* ptr,
                                                  size_t num_data_bytes_in,
-                                                 int32_t offset,
-                                                 int idx,
+                                                 size_t offset,
+                                                 size_t idx,
```

Comment:
when you change `int` to `size_t` ensure that it is safe in many places in code negative values is used to find issue or special case - in such case you have unexpected behavior if you lost the sign.